backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/UpdateVmCommand.java

```diff
@@ -1397,6 +1397,11 @@ && isHotSetEnabled()
                 getParameters().isUpdateNuma(), getEffectiveCompatibilityVersion()))) {
             return false;
         }
+
+        if (getParameters().getVmStaticData().getConsoleDisconnectActionDelay() < 0) {
+            return failValidation(EngineMessage.ACTION_TYPE_FAILED_ILLEGAL_NUM_OF_MONITORS);
```

Comment:
you meant to return the new error ACTION_TYPE_FAILED_ILLEGAL_CONSOLE_DISCONNECT_ACTION_DELAY, right?